content/markdown/02-working-with-data/02-04-grouping-summarizing.rmd

```diff
@@ -0,0 +1,70 @@
+---
+output:
+  pdf_document: default
+  html_document: default
+---
+# Grouping & Summarizing DataFrames
```

Comment:
I may have missed this earlier- but is a tibble ever explained? If not, might be good to add in one of the earlier pages